contracts/index/oracles/OracleArbitrum.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import { ContextUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+
+import { IChainlinkAggregatorV3 } from "../dependencies/IChainlinkAggregatorV3.sol";
+import { IIndexOracle } from "../interfaces/IIndexOracle.sol";
+import { Errors } from "../libraries/Errors.sol";
+
+contract OracleArbitrum is
```

Comment:
small thing, but is there anything making this Oracle 'Arbitrum' specific? If not, then we can just use a more general name like ChainlinkOracleWrapper or something like that.

Reply:
When we support AMM price, OracleArbitrum will support Uniswap V3 while OracleAvalanche won't.

Reply:
We could also have only one oracle for all chains supporting everything, but during configuration (when we specify which AMMs are supported) we configure them based on the chain. It is not that important, so if you changed the URL, I can approve.